Disambiguated text.txt

```diff
@@ -0,0 +1,1582 @@
+	"main" adj
+"<processing>"
+	"processing" n sg
+;	"process" vblex pprs
+;	"process" vblex subs
+;	"process" vblex ger
+"<steps in>"
+	"step# in" vblex pres p3 sg
+"<a>"
+	"a" det ind sg
+"<computer>"
+;	"computer" adj
+	"computer" n sg
+"<:>"
+	":" sent
+"<inputting>"
+	"inputting" vblex ger
+"<,>"
+	"," cm
+"<storage>"
+	"storage" n sg
+"<,>"
+	"," cm
+"<outputting>"
+;	"output" vblex pprs
+;	"output" vblex subs
+	"output" vblex ger
+"<and>"
+	"and" cnjcoo
+"<processing>"
+;	"processing" n sg
+;	"process" vblex pprs
+;	"process" vblex subs
+	"process" vblex ger
+"<..>"
+	".." sent
+
+"<Modern>"
+	"modern" adj
+"<computers>"
+	"computer" n pl
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<do>"
+;	"do" vblex inf
+;	"do" vblex pres
+;	"do" vblex imp
+;	"do" vaux inf
+;	"do" vaux pres
+	"do" vbdo inf
+;	"do" vbdo pres
+;	"do" vbdo imp
+"<billions>"
+	"billion" n pl
+"<of>"
+	"of" pr
+"<calculations>"
+	"calculation" n pl
+"<in a second>"
+	"in a second" adv
+"<.>"
+	"." sent
+"<Being able to>"
+	"be# able to" vbser pprs
+;	"be# able to" vbser subs
+;	"be# able to" vbser ger
+"<calculate>"
+	"calculate" vblex inf
+;	"calculate" vblex pres
+;	"calculate" vblex imp
+"<many times>"
+	"many times" adv
+"<per>"
+	"per" pr
+"<second>"
+;	"second" adv
+	"second" n sg
+;	"second" adj ord
+;	"second" vblex inf
+;	"second" vblex pres
+;	"second" vblex imp
+"<allows>"
+	"allow" vblex pres p3 sg
+"<modern>"
+	"modern" adj
+"<computers>"
+	"computer" n pl
+"<to>"
+	"to" pr
+"<multitask>"
+	"multitask" vblex inf 
+"<,>"
+	"," cm
+"<which>"
+;	"which" det itg sp
+;	"which" prn itg mf sp
+	"which" prn rel an mf sp
+"<means>"
+;	"mean" n pl
+;	"means" n sg
+;	"means" n pl
+	"mean" vblex pres p3 sg
+"<they>"
+	"prpers" prn subj p3 mf pl
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<do>"
+;	"do" vblex inf
+;	"do" vblex pres
+;	"do" vblex imp
+;	"do" vaux inf
+;	"do" vaux pres
+	"do" vbdo inf
+;	"do" vbdo pres
+;	"do" vbdo imp
+"<many>"
+	"many" det qnt pl
+;	"many" prn ind mf pl
+"<different>"
+	"different" adj
+"<tasks>"
+	"task" n pl
+;	"task" vblex pres p3 sg
+"<at the same time>"
+	"at the same time" adv
+"<.>"
+	"." sent
+"<Computers>"
+	"computer" n pl
+"<do>"
+;	"do" vblex inf
+;	"do" vblex pres
+;	"do" vblex imp
+;	"do" vaux inf
+;	"do" vaux pres
+;	"do" vbdo inf
+	"do" vbdo pres
+;	"do" vbdo imp
+"<many>"
+	"many" det qnt pl
+;	"many" prn ind mf pl
+"<different>"
+	"different" adj
+"<jobs>"
+	"job" n pl
+"<where>"
+	"where" adv
+;	"where" adv itg
+"<automation>"
+	"automation" n sg
+"<is>"
+	"be" vbser pres p3 sg
+"<useful>"
+	"useful" adj
+"<.>"
+	"." sent
+"<Some>"
+;	"some" adj
+;	"some" adv
+	"some" det qnt sp
+;	"some" prn ind mf pl
+"<examples>"
+	"example" n pl
+"<are>"
+	"be" vbser pres
+"<controlling>"
+;	"controlling" n sg
+	"control" vblex pprs
+;	"control" vblex subs
+;	"control" vblex ger
```

Comment:
although this follows a copula, it's not a progressive construction (it could instead have been "examples include controlling traffic lights ...") so actually `<subs>`